packages/server/src/seeder/utils.ts

```diff
@@ -13,13 +16,17 @@ export async function markdownToJSON() {
 /**
  * Parse the Markdown to HTML
  */
-export async function markdownToHTML() {
-    // Markdown file is located in the assets folder
```

Comment:
You should not delete the comments as it describes the purpose and process of this function. If deleted, you should add your own comments.